.yarn/versions/0a63a0fb.yml

```diff
@@ -0,0 +1,2 @@
+releases:
+  "@ecocode/eslint-plugin": minor
```

Comment:
why this file didn't exist before this PR with the already existing rule ?

Reply:
Existing rule has already been published into [v0.1.0](https://github.com/green-code-initiative/ecoCode-linter/releases/tag/eslint-plugin%2F0.1.0), so we should tell which semantic version this rule will need (this a new feature, but not a breaking change, so minor seems to be good here)